Mechanisms_evolution/Mechanisms_Evolution.Rmd

```diff
@@ -9,252 +9,222 @@ output:
 knitr::opts_chunk$set(echo = TRUE,fig.width = 5, fig.height = 3)
 ```
 
-## 1. Getting Started
+## Goals for Lab 2
+
+* Write commands based on descriptive comments
+* Subset your data using dplyr
+* Use R to calculate the mean of subsets of data
+* Plot line graphs in ggplot
+* Use colors to show difference subsets of data
+* Perform a t-test
+
 
-**Make a new project in a new folder and open a new script.**
+## 1. Getting Started
 
-It is standard practice to first load all libraries you will need. In this case, we will be using `ggplot2` again to make our graphs. We are also entering data into Google Sheets, so we will need `gsheet` to pull that data into R. 
+**Make a new project and open a new script.**
 
-For this lab we will provide you with the directions of what to do by showing you the comments you should enter in you R script.
+Begin by loading all the libraries you need. In this case, we will be using `ggplot2` to make our graphs. We are also entering data into Google Sheets, so we will need `gsheet` to pull that data into R. 
 
 * Copy the comment lines into your script
-* Enter the necessary code based on the previous lab. In any case where you have not seen the necessary code it is provided. **In cases where you should know the code already you should check assignment 1 if you need help.**
+* Enter the necessary code based on the previous lab and your R script from Lab 1. **In cases where you should know the code already you should check the Lab 1 Manual if you need help.**
 
 ```{r load1}
-# Load the libraries ggplot2 and gsheet
+# Load the packages ggplot2, gsheet, and dplyr
```

Comment:
I've been calling them packages (which they are) but they are loaded with library() - is that confusing?

Reply:
Yes! We've been saying that they are packages, and the command to load them is library()

Reply:
refer to packages - it's ok to load as library